back/spec/acceptance/ideas/ideas_create_spec.rb

```diff
@@ -746,7 +746,27 @@ def public_input_params(context)
         example_request '[Unauthorized] Cannot create a common-ground input' do
           assert_status 401
           expect(json_response_body)
-            .to match(errors: { base: [{ error: 'posting_not_supported' }] })
+            .to match(errors: { base: [{ error: 'Unauthorized!' }] })
```

Comment:
Should we add a context like "when phase authentication requirements is set to 'none'", as I assume in that case the request should succeed.